source/configuration_manual/mail_location/index.rst

```diff
@@ -129,34 +129,32 @@ Key                Value Description
                    directories are stored. The default is empty unless
                    otherwise described in the mailbox format pages.
 
+``DIRNAME``        Specifies the directory name used for mailbox directories,
+                   or in the case of mbox specifies the mailbox message file
+                   name.
```

Comment:
This could further say: `sdbox and mdbox formats use DIRNAME=dbox-Mails by default.`